testplan/common/utils/comparison.py

```diff
@@ -317,6 +319,105 @@ def compare_with_callable(callable_obj, value):
         return False, format_trace(inspect.trace(), exc)
 
 
+class RE(object):
```

Comment:
pyom is doing deepcopy of RE object, that's why we need it.

Reply:
Why is pyom doing that for? I think we should fix that, rather than adding a hacky workaround here.

Reply:
(Sorry if my review is inappropriate, my assumption is that code reviews are public because outsider comments are welome.)

I think pyom should be fixed to properly deepcopy `Pattern` objects, and you only need RE to be a function alias for `re.compile` for internal compatibility. AFAIK internal user code mainly doesn't depend on RE being a standalone (wrapper) class.